CONTRIBUTING.md

```diff
@@ -62,9 +62,11 @@ For more, because etcd uses Kubernetes' prow infrastructure to run CI jobs, the
 If you find any flaky tests on testgrid, please
 
 1. Check [existing issues](https://github.com/etcd-io/etcd/issues?q=is%3Aissue+is%3Aopen+label%3Atype%2Fflake) to see if an issue has already been opened for this test. If not, open an issue with the `type/flake` label.
-2. Try to reproduce the flaky test on your machine via `stress`, for example, to reproduce the failure of `TestPeriodicSkipRevNotChange`:
+2. Try to reproduce the flaky test on your machine via `stress` (https://pkg.go.dev/golang.org/x/tools/cmd/stress), for example, to reproduce the failure of `TestPeriodicSkipRevNotChange`:
 
 ```bash
+# install the stress utility
```

Comment:
This code snippet below does not work as expected. To clarify, your contribution looks good, but could you help us correct the second part?

Compiling the tests `go test -c` will compile a binary with all the tests, not just the one from the matching regular expression. I noticed this in the instructions we had in our etcd triage meeting, and I updated the snippet from there to address the issue (https://docs.google.com/document/d/16XEGyPBisZvmmoIHSZzv__LoyOeluC5a4x353CX0SIM/edit?pli=1#heading=h.e48w90nud901).

We need first to compile and then run stress passing the regular expression:

```bash
go test -v -c -count 1
stress -p=8 ./v3compactor.test -test.run “^TestPeriodicSkipRevNotChange$”
```